test/link-check.mjs

```diff
@@ -6,29 +6,38 @@ import * as urlModule from 'url'
 import markdownLinkCheck from 'markdown-link-check'
```

Comment:
It's a good idea to enforce style on the files in `test/`, but if we do so we should probably go whole hog and convert them to TypeScript formatted with GTS. It would be really weird to hold them to a style that's different from the one used in the rest of the repo.

Reply:
I tried converting these to TS but it added a bunch of brittle machinery (declaration types missing for some packages, ts-node choking on ESM https://github.com/TypeStrong/ts-node/issues/1007). So instead I converted these to JS, and included them in the directories formatted by GTS.